modules/ROOT/pages/apikit-odatav4-transform-to-sql-operation.adoc

```diff
@@ -0,0 +1,152 @@
+= Transforming to SQL Query
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can transform the input OData `GET` request to a SQL SELECT query with the `Transform to SQL Select` operation. This operation is useful when combined with the `select` operation of the Database Connector. 
```

Comment:
```suggestion
You can transform the input OData `GET` request to an SQL SELECT query by using the `Transform to SQL Select` operation. This operation is useful when combined with the `select` operation of Database Connector. 
```

Reply:
Always spell out the connector name on first mention.

Reply:
OK! Thanks!